pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -491,14 +496,59 @@ func shouldIgnore(a admission.Attributes) (bool, error) {
 	if pod.Spec.OS != nil && pod.Spec.OS.Name == coreapi.Windows {
 		return true, nil
 	}
-	// if this is an update, see if we are only updating the ownerRef.  Garbage collection does this
+
+	if a.GetOperation() == admission.Update {
+		oldPod, ok := a.GetOldObject().(*coreapi.Pod)
+		if !ok {
+			return false, admission.NewForbidden(a, fmt.Errorf("object was marked as kind pod but was unable to be converted: %v", a.GetOldObject()))
+		}
+
+		// never ignore any spec changes
```

Comment:
once vertical scaling is a thing, we will ignore some. Just be ready.

Reply:
is https://github.com/openshift/apiserver-library-go/pull/128 legit, too?